src/Symfony/Component/HttpKernel/Tests/Fixtures/BundleDependencies/BundleCDependenciesBA.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Tests\Fixtures\BundleDependencies;
+
+use Symfony\Component\HttpKernel\Bundle\BundleDependenciesInterface;
+
+class BundleCDependenciesBA implements BundleDependenciesInterface
+{
+    public function getBundleDependencies()
+    {
+        return array(
+            'Symfony\Component\HttpKernel\Tests\Fixtures\BundleDependencies\BundleBDependenciesA',
```

Comment:
if we are to ever add optional support imho it would be better to have an associative array here, where the value would be true for required and false for optional for example. then when doing the class_exists check you would simply not throw an exception for optional dependencies.